assets/app.js

```diff
@@ -58,6 +58,16 @@ $(function () {
         updateSelect2Options(e.target.value);
         $('.engine-options').addClass('hidden');
         $(`#${e.target.value}-options`).removeClass('hidden');
+        let engine = e.target.value;
+        if(engine === 'tesseract' || engine === 'google') {
+            $('#lang').removeAttr('required');
```

Comment:
`.prop('required', false)` should be used to unset the `required` attribute, see https://api.jquery.com/prop/#prop2